apis/sgv2-restapi/src/main/java/io/stargate/sgv2/restapi/service/resources/schemas/Sgv2KeyspacesResourceImpl.java

```diff
@@ -100,8 +109,9 @@ public Uni<RestResponse<Sgv2NameResponse>> createKeyspace(final String payloadSt
               .ifNotExists()
               .withReplication(Replication.simpleStrategy(ksCreateDef.replicas))
               .build();
+      dcMap = Collections.emptyMap();
```

Comment:
is this option tested now with empty map?

Reply:
There is an existing test for simple creation so this code path is covered to a degree -- but empty map is just used for convenience so that `failureMatchesBadDC()` need not to check special case.